homeassistant/components/opentherm_gw/__init__.py

```diff
@@ -431,3 +431,14 @@ async def handle_report(status):
             async_dispatcher_send(self.hass, self.update_signal, status)
 
         self.gateway.subscribe(handle_report)
+
+
+class MultilineListFilter(logging.Filter):
```

Comment:
Can we use `pprint.pformat` instead?
https://github.com/home-assistant/core/blob/6d043f2ca12ca6e218c86e168f40da4fe5348fd2/homeassistant/components/deconz/config_flow.py#L95

Reply:
Done, and tests are passing again.